ott/tools/plot.py

```diff
@@ -107,6 +126,41 @@ def barycenters(arg: Union[transport.Transport, jnp.ndarray],
 
   if isinstance(arg, transport.Transport):
     ot = arg
-    return _barycenters(ax, ot.geom.y, ot.a, ot.b, ot.matrix, **kwargs)
-
-  return _barycenters(ax, arg, a, b, matrix, **kwargs)
+    _barycenters(ax, ot.geom.y, ot.a, ot.b, ot.matrix, **kwargs)
+    return ax
+
+  _barycenters(ax, arg, a, b, matrix, **kwargs)
+  return ax
+
+
+def animate(image_fns: List[str],
+            gif_fn: str,
+            duration: int = 250,
+            loop: int = 0):
+  """Makes an animated GIF from list of images."""
+  if not PIL_is_importable:
+    warnings.warn(
+        'Pillow was not imported successfully. Doing nothing instead.',
+        RuntimeWarning,
+    )
+    return
+  imgs = [Image.open(_) for _ in image_fns]
```

Comment:
I would rather not use the underscore for arguments that are being used.

Reply:
Also you are using PIL only for opening images, this is unnecessary. Use plt.imread instead and remove the cumbersome dependencies on PIL

Reply:
underscore fixed.

Reply:
PIL: see comments

Reply:
Did you add a comment in the code ? i am not sure what you are referring to.

Reply:
Oh I mean the comments below in the github code review.